XenModel/Messages.resx

```diff
@@ -7270,7 +7270,10 @@ This might result in failure to migrate VMs to this server during the RPU or to
     <value>Place &amp;all imported virtual disks on this target SR:</value>
   </data>
   <data name="IMPORT_WIZARD_CPUS_COUNT_MISMATCH_HOST" xml:space="preserve">
-    <value>The imported appliance requires a minimum of {0} vCPUs, while the number of physical CPUs in the selected server is {1}. You will not be able to start the appliance on the selected server.</value>
+    <value>The imported appliance requires a minimum of {0} vCPUs on at least one VM, but there are servers with insufficient physical CPUs ({1}). Ensure that the VMs are mapped correctly or you will not be able to start the appliance on the selected server.</value>
+  </data>
+  <data name="IMPORT_WIZARD_CPUS_COUNT_MISMATCH_HOST_ALL" xml:space="preserve">
+    <value>The imported appliance requires a minimum of {0} vCPUs, while the maximum number of physical CPUs in the selected servers is {1}. You will not be able to start all the VMs in the appliance with the current selection.</value>

```

Comment:
I think the fist part of this one too needs to be rephrased to `At least one VM in the imported appliance requires a minimum of {0} vCPUs, while...`